pymatviz/ptable.py

```diff
@@ -699,292 +730,438 @@ def histogram(
         ax.spines[["right", "top", "left"]].set_visible(False)
 
 
-def ptable_heatmap(
-    values: ElemValues,
-    *,
-    log: bool | Normalize = False,
-    ax: plt.Axes | None = None,
-    count_mode: ElemCountMode = ElemCountMode.composition,
-    cbar_title: str = "Element Count",
-    cbar_range: tuple[float | None, float | None] | None = None,
-    cbar_coords: tuple[float, float, float, float] = (0.18, 0.8, 0.42, 0.05),
-    cbar_kwargs: dict[str, Any] | None = None,
-    colorscale: str = "viridis",
-    show_scale: bool = True,
-    show_values: bool = True,
-    infty_color: str = "lightskyblue",
-    na_color: str = "white",
-    heat_mode: Literal["value", "fraction", "percent"] | None = "value",
-    fmt: str | Callable[..., str] | None = None,
-    cbar_fmt: str | Callable[..., str] | None = None,
-    text_color: str | tuple[str, str] = "auto",
-    exclude_elements: Sequence[str] = (),
```

Comment:
This arg `exclude_element` is now removed, the rationale being: I think it's something that should be controlled by the `data` itself. I.e. if an element is not present in the data, then it would not be included.